app/models/salon.rb

```diff
@@ -2,4 +2,10 @@ class Salon < ApplicationRecord
   has_many :products, dependent: :destroy
   has_many :comments
   belongs_to :owner, class_name: 'User'
+
+  def self.update_salon_rating(salon)
```

Comment:
It's better to call this method from a salon instance, like Salon.find(id).update_salon_rating